core/src/test/scala/org/apache/spark/memory/StaticMemoryManagerSuite.scala

```diff
@@ -43,12 +43,13 @@ class StaticMemoryManagerSuite extends MemoryManagerSuite {
 
   override protected def createMemoryManager(
       maxOnHeapExecutionMemory: Long,
-      maxOffHeapExecutionMemory: Long): StaticMemoryManager = {
+      maxOffHeapExecutionMemory: Long = 1000): StaticMemoryManager = {
     new StaticMemoryManager(
       conf.clone
         .set("spark.memory.fraction", "1")
         .set("spark.testing.memory", maxOnHeapExecutionMemory.toString)
-        .set("spark.memory.offHeap.size", maxOffHeapExecutionMemory.toString),
+        .set("spark.memory.offHeap.size",
```

Comment:
How about move the configuration to `internal/config`?

Reply:
I see. I will move `"spark.memory.offHeap.size"` to `org.apache.spark.internal`.

Reply:
To add `= 1000` is intentional and necessary.  
If the original line is used, `spark.memory.offHeap.size` is zero even in the case `spark.memory.offHeap.enabled == true`. This is because the caller sites do not pass the second argument and then the default value `(= 0)` is used.

Reply:
we can use `config.MEMORY_OFFHEAP_SIZE.key` here.